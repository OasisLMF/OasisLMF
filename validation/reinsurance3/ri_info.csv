ReinsNumber,ReinsName,ReinsPeril,ReinsInceptionDate,ReinsExpiryDate,CededPercent,RiskLimit,RiskAttachment,OccLimit,OccAttachment,ReinsCurrency,InuringPriority,ReinsType,AttachmentBasis,PlacedPercent,DeemedPercentPlaced,ReinsFXrate,TreatyShare,UseReinsDates,RiskLevel
1,Cedant1,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
2,Producer2,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
3,RES,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
4,GB,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
5,ReinsTag5,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
6,LocGroup6,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
7,Cedant7 and COM,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
8,IND and IE,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
9,COM and ReinsTag9,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
10,All Loc 10 filters,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
11,Cedant11 or Producer12,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
12,AGR or AT,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
13,LocGroup15 or ReinsTag16,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
14,Cedant17 and IT or IT and ReinsTag18,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
15,Port4 Loc19 or CH,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
101,ReinsNumber1_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
102,ReinsNumber2_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
103,ReinsNumber3_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
104,ReinsNumber4_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
105,ReinsNumber5_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
106,ReinsNumber6_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
107,ReinsNumber7_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
108,ReinsNumber8_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
109,ReinsNumber9_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
110,ReinsNumber10_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
111,ReinsNumber11_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
112,ReinsNumber12_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
113,ReinsNumber13_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
114,ReinsNumber14_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
115,ReinsNumber15_ctrl,WW1,01/01/2024,31/12/2024,1,0,0,0,0,GBP,1,QS,LO,0.5,1,1,1,N,
