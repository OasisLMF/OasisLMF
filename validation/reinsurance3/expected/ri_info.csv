ReinsNumber,ReinsName,ReinsPeril,ReinsInceptionDate,ReinsExpiryDate,CededPercent,RiskLimit,RiskAttachment,OccLimit,OccAttachment,ReinsCurrency,InuringPriority,ReinsType,AttachmentBasis,PlacedPercent,DeemedPercentPlaced,ReinsFXrate,TreatyShare,UseReinsDates,RiskLevel
1,Cedant1,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
2,Producer2,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
3,RES,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
4,GB,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
5,ReinsTag5,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
6,LocGroup6,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
7,Cedant7 and COM,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
8,IND and IE,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
9,COM and ReinsTag9,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
10,All Loc 10 filters,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
11,Cedant11 or Producer12,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
12,AGR or AT,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
13,LocGroup15 or ReinsTag16,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
14,Cedant17 and IT or IT and ReinsTag18,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
15,Port4 Loc19 or CH,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
101,ReinsNumber1_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
102,ReinsNumber2_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
103,ReinsNumber3_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
104,ReinsNumber4_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
105,ReinsNumber5_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
106,ReinsNumber6_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
107,ReinsNumber7_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
108,ReinsNumber8_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
109,ReinsNumber9_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
110,ReinsNumber10_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
111,ReinsNumber11_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
112,ReinsNumber12_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
113,ReinsNumber13_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
114,ReinsNumber14_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
115,ReinsNumber15_ctrl,WW1,01/01/2024,31/12/2024,1.0,0.0,0.0,0.0,0.0,GBP,1,QS,LO,0.5,1.0,1.0,1.0,N,SEL
